### DISCUSSION TITLE
How to change "click to agree" prompt text?

### DISCUSSION THREAD

**Discussion by WillMorhler** [mastodon/mastodon#21576] [Q&A]
Is there a way to change the text (including the link text) that appears next to the "check here to agree" box on the new user sign up page? (The legal department of my organization wants it read something like "Terms of Service" rather than "Privacy Policy.")

(I understand that we can include our own text as the body of that document, looking here to change what the user sees on the signup page.)

Currently running a Digital Ocean 1-click droplet, upgraded to version 4.0.1.

We're not looking to make other configuration changes or web design modification; I just  assume there is a file containing this text somewhere that can be edited.

Thanks!

**Comment by mjankowski** [comment]
I realize you asked this question three years ago, but -- there's no simple configuration for this from the admin UI, but if you are comfortable making code changes this would probably be as simple as editing an i18n strings file.

Separatley - there is a (still somewhat WIP) feature to actually have a separate editable TOS feature, which (when in place) would somewhat automatically put that text into the prompt area in question.
